### PR TITLE
Release notes update for token signatures

### PR DESCRIPTION
**Pull request by stianst**

### CODE REVIEW

File: release_notes/topics/4_5_0_final.adoc [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+Thanks to https://github.com/tnorimat[tnorimat] for contributing a signficant part of this work.
+
+= New Signature Algorithms
+
+Alongside the Signature SPI there is now also support for additional signature algorithms.
+
+Keycloak now has support for RS256, RS384, RS512, ES256, ES384, ES512, HS256, HS384 and HS512.
+
+Elliptic Curve Digital Signature Algorithm (ES256/384/512) are very interesting as they provide similar
+security properties as RSA signatures, but uses significantly less CPU.
```
Comment (matthewhelmke) [inline release_notes/topics/4_5_0_final.adoc]: Should be "...but use significantly less..."

File: release_notes/topics/4_5_0_final.adoc [hunk 2]
```diff
@@ -0,0 +1,22 @@
+= Signature SPI
+
+The Signature SPI makes it possible to plug-in additional signature algorithms. Not only does it enable
+additional signatures, but it also enables changing how signatures are generated. For example this would allow
+using an HSM device to sign tokens.
+
```
Comment (matthewhelmke) [inline release_notes/topics/4_5_0_final.adoc]: The Signature SPI makes it possible to plug-in additional signature algorithms. This enables additional signatures and also enables changing how signatures are generated. For example, using this allows using an HSM device to sign tokens.

**Review by matthewhelmke — CHANGES_REQUESTED** [review]
Looks good. I have a couple edit suggestions for greater effect.

**Comment by stianst** [comment]
Updated suggestions and merged to make sure it's included in the 4.5 release

**Pull request merged**